db/types.go

```diff
@@ -61,8 +61,7 @@ type DBConnection interface {
 	// Deletes a saved version. Returns ErrVersionDoesNotExist for invalid versions.
 	DeleteVersion(uint64) error
 
-	// Reverts the DB state to the last saved version.
-	// Returns an error if no saved versions exist.
+	// Reverts the DB state to the last saved version; if none exist, this clears the DB.
```

Comment:
nit: Let's abide by godoc standard and make these (and other) comments begin with the method/function/type/variable name. Related, most the methods in memdb/db.go and badgerdb/db.go are missing godoc comments.